.bazelrc

```
@@ -5,3 +5,10 @@ query --deleted_packages=examples/cc/external,examples/command_line/external,exa
 
 # Import Shared settings
 import %workspace%/shared.bazelrc
+
+# Use a local.bazelrc if it exists.
+try-import %workspace%/local.bazelrc
```

```suggestion
try-import %workspace%/user.bazelrc
```

I think some bazel docs recommend calling this `user.bazelrc`.